tests/test_grid3d/test_grid.py

```diff
@@ -798,3 +798,16 @@ def test_get_vtk_geometries_emerald(show_plot):
         plt.set_scale(1, 1, 5)
         plt.show_axes()
         plt.show()
+
+
+def test_get_vtk_esg_geometry_data_smoketest(show_plot):
+    grd = xtgeo.create_box_grid((2, 6, 4), increment=(20, 10, 7))
+    grd._actnumsv[0, 0, 0] = 0
```

Comment:
Give a docstring to explain the test. And is `show_plot` fixture applied? Also consider other tests variants?

Reply:
Renamed existing test and added a more elaborate test that inspects the actual vertex coordinates returned.
Added docstrings and removed reference to `show_plot` fixture